src/Orleans.Core.Abstractions/Placement/PlacementStrategy.cs

```diff
@@ -16,5 +22,14 @@ public abstract class PlacementStrategy
         /// If true then activations have activation ids equal to their grain id, otherwise activations are given unique ids.
         /// </summary>
         internal virtual bool IsDeterministicActivationId => false;
+
+        public virtual void Initialize(GrainProperties properties)
+        {
+        }
+
+        public virtual void PopulateGrainProperties(IServiceProvider services, Type grainClass, GrainType grainType, Dictionary<string, string> properties)
+        {
+            properties[WellKnownGrainTypeProperties.PlacementStrategy] = this.GetType().Name;
```

Comment:
Do you see a use case where we would never set this property? Not a lot of people will implement their own `PlacementStrategy`, but I assume it will be easy to overlook calling the base method.

Not very important anyway.

Reply:
Yeah, it's definitely possible. If a developer wants to override `PopulateGrainProperties` then that's fine, but caveat emptor, I suppose.

The stateless worker overrides this and adds another property